### ISSUE TITLE
Why does learning rate (lr)  increase in function "warmup"?

### ISSUE THREAD

**Issue by cuttle-fish-my** [a6o/3d-diffusion-pytorch#3]
I've noticed that in `train.py` and function `warmup`, the learning rate increases proportionally with `step`, which immensely confuses me because the 'lr' should usually decrease along the training process.

**Comment by a6o** [comment]
linear warmup is a technique of increasing LR from 0 to some value for when the model is starting to be trained. For diffusion models, this seem to help alot.  In my experience, it also helped to have more consistent results. 

Use of linear warmup is explained in the paper, in section 7.2 (supplementary).

I am not the original author of this paper, and unfortunately, I was never able to get the same results as the paper.
